src/ruby/puppet-server-lib/puppet/server/certificate.rb

```diff
@@ -15,6 +16,7 @@ def initialize(java_cert)
     end
 
     @java_cert = java_cert
+    @oid_defns = Puppet::Server::Oids.load_custom_oid_file(Puppet[:trusted_oid_mapping_file]) + Puppet::SSL::Oids::PUPPET_OIDS
```

Comment:
Further, if we load the custom OID file from Ruby, doesn't that mean that there will be no clean way for @nathanielksmith to access it for TK auth unless he duplicates the parsing logic?  Maybe we feel like the parsing logic is simple enough to tolerate the duplication?